charts/polymorphic-app/templates/worker.yaml

```diff
@@ -175,15 +195,27 @@ spec:
 apiVersion: autoscaling/v2beta1
 kind: HorizontalPodAutoscaler
```

Comment:
Should we add a condition not to add HPA for StatefulSets?

Reply:
we can't create StatefulSet for worker, so there no need to add condition in worker.yaml